Tools/LambdaTestTool/src/Amazon.Lambda.TestTool/Amazon.Lambda.TestTool.csproj

```diff
@@ -9,7 +9,9 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Amazon.Lambda.APIGatewayEvents" Version="2.5.0" />
```

Comment:
Bonus challenge in this tool is you can't take dependencies like `Amazon.Lambda.APIGatewayEvents` because it would then interfere with the version the Lambda function is depending on.